src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,105 @@
+import styles from "./Dropdown.module.scss";
+import dropdownIcon from "../../assets/images/dropdownArrowIcon.svg";
+import { FC, useEffect, useRef, useState } from "react";
+import Rating from "../Rating/Rating";
+interface DropDownProps {
+  filterTitle: string;
+  rating?: boolean;
+  priceRange?: boolean;
+  distance?: boolean;
+  setPriceRange?: React.Dispatch<React.SetStateAction<number[]>>;
+  setRating?: React.Dispatch<React.SetStateAction<number>>;
+}
+
+const Dropdown: FC<DropDownProps> = (props) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [isChecked, setIsChecked] = useState([
+    false,
+    false,
+    false,
+    false,
+    false,
+  ]);
+  const ratingPopoverRef = useRef<HTMLDivElement | null>(null);
+
+  const hasAtLeastOneTrue = (booleanArray: boolean[]) => {
+    return booleanArray.some((value) => value === true);
```

Comment:
You can just write `booleanArray.some((value) => value)` as the `some()` method will automatically check for the boolean value of the returned value.

Reply:
@nitzanpap  done